package.json

```diff
@@ -27,37 +27,55 @@
     "Google Cloud Vision API"
   ],
   "scripts": {
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "lint": "gts fix && eslint --fix samples/*.js",
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js smoke-test/*.js --timeout 600000",
+    "system-test": "mocha build/system-test",
+    "fix": "gts fix",
     "test-no-cover": "mocha test/*.js",
-    "test": "nyc mocha",
-    "fix": "eslint --fix '**/*.js'",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
+    "prelint": "cd samples; npm link ../; npm i",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "@google-cloud/promisify": "^1.0.0",
-    "google-gax": "^1.7.5",
-    "is": "^3.2.1"
+    "@google-cloud/promisify": "^1.0.4",
+    "@types/is": "0.0.21",
+    "@types/sinon": "^7.5.1",
+    "@types/uuid": "^3.4.7",
+    "chai": "^4.2.0",
```

Comment:
We should remove `chai`

Reply:
Here we use `assert.match` from `const assert = require('chai').assert;` in `helpers.test.ts`

Reply:
Several more years and we'll be able to use it without `chai` :)

![image](https://user-images.githubusercontent.com/4015807/75193037-4fe63d80-570a-11ea-8122-f0ca1ae73c62.png)

Maybe just do `assert("abcde".match(/zzz/))` and stop using `chai`? It will give less fancy error message, but other than that, will work perfectly.

Reply:
I tried directly using `assert.match(value, regex)` and it complains about `assert.match is not a function` but I will dig into more. Thank you

Reply:
removed `chai` dependency, thanks